src/grid/lebedev.py

```diff
@@ -117,12 +118,12 @@ def generate_lebedev_grid(*, degree=None, size=None):
         An AngularGrid instance with points and weights.
     """
     degree, size = _select_grid_type(degree=degree, size=size)
-    points, weights = _load_grid_arrays(_load_grid_filename(degree, size))
+    points, weights = _load_grid_filename(degree, size)
     # set weights to 4\pi
     return AngularGrid(points, weights * 4 * np.pi)
 
 
-def match_degree(degree_nums):
+def match_degree(degree_nums):  # clear way coding
```

Comment:
If this is not addressed in this PR, please change `# clear way coding` to `# TODO: clear way coding`, so it is not easier to track and address later.